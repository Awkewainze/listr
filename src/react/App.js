import React from 'react';
import './App.scss';
import Header from './common/Header';
import Footer from './common/Footer';
import { Switch, Route } from 'react-router-dom';
import ActiveListContainer from './list/ActiveListContainer';

function App() {
  return (
    <div className="App">
      <Header/>
      <Switch>
        <Route path="/" component={ActiveListContainer} />
      </Switch>
      <Footer/>
    </div>
  );
}

export default App;
