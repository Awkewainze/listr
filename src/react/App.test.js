import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import App from './App';

it('matches snapshot', () => {
  expect(toJson(shallow(<App/>))).toMatchSnapshot();
});