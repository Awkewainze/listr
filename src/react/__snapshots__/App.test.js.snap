// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="App"
>
  <Layout>
    <Switch>
      <Route
        component={[Function]}
        path="/"
      />
    </Switch>
  </Layout>
</div>
`;
